public/pyscript/main.py

```
@@ -1,101 +1,114 @@
-            # file_data = file.arrayBuffer().to_py()
-            # img = Image.open(BytesIO(file_data))
-            # img_with_border = ImageOps.expand(img, border=bleed_amount, fill="red")
-            # img_with_border.show()
-            console.log(f"✅ File {i + 1}/{len(files)}: {file.name} processed successfully")
-        console.log("🎉 All images processed successfully!")
-    except Exception as e:
-        console.error(f"❌ Error processing images: {str(e)}")
+from PIL import Image
+from io import BytesIO
+
+from pyscript import when # noqa # pyright: ignore [reportAttributeAccessIssue]
```

Ruff (0.11.9)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)